virt/virt-4-8-release-notes.adoc

```diff
@@ -94,7 +94,8 @@ Deprecated features are included in the current release and supported. However,
 [id="virt-4-8-changes"]
 == Notable technical changes
 
-//CNV-9052 OpenShift Virtualization now automatically automatically configures IPv6 IP when running on dual-stack clusters.
+//CNV-9052 OpenShift Virtualization now automatically configures IPv6 IP when running on dual-stack clusters.
```

Comment:
Maybe worth to use same semantic IPv6 addresses as next line instead IPv6 IP ?

Reply:
This is just a placeholder comment and is not rendered in the published output